core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamTest.scala

```diff
@@ -21,6 +21,11 @@ import org.scalatest.{Matchers, FunSuite}
 class SimpleDateParamTest extends FunSuite with Matchers {
 
   test("date parsing") {
+    import java.util.{Locale, TimeZone}
```

Comment:
> I'm slightly afraid this will have negative effects on other tests since you're setting the defaults globally for the JVM.

seems in spark all the date related tests use America/Los_Angeles time zone, so i think it will be ok.

> Why would this fail though if the date parameter specifies its time zone? it should not be ambiguous.

I also do not get the root cause. but actually it is time zone sensitive, we can have a simple test for this, after we specify the timezone other than "America/Los_Angeles", this test will fail:
```
// Timezone is fixed to America/Los_Angeles for those timezone sensitive tests (timestamp_*)
      TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
      // Add Locale setting
      Locale.setDefault(Locale.US)
     new SimpleDateParam("2015-02-20T23:21:17.190GMT").timestamp should be (1424474477190L)
     new SimpleDateParam("2015-02-20T17:21:17.190CST").timestamp should be (1424474477190L)
     new SimpleDateParam("2015-02-20").timestamp should be (1424390400000L)  // GMT
```